src/main/scala/xiangshan/mem/lsqueue/LoadQueue.scala

```diff
@@ -119,6 +119,7 @@ class LoadQueue(implicit p: Parameters) extends XSModule
   val released = RegInit(VecInit(List.fill(LoadQueueSize)(false.B))) // load data has been released by dcache
   val error = RegInit(VecInit(List.fill(LoadQueueSize)(false.B))) // load data has been corrupted
   val miss = Reg(Vec(LoadQueueSize, Bool())) // load inst missed, waiting for miss queue to accept miss request
+  val need_refill_by_dcache = Reg(Vec(LoadQueueSize, Bool())) // load inst missed, the data should be refilled by dcache
```

Comment:
Is `need_refill_by_dcache` necessary? Is it possible to use `datavalid` for this?